### ISSUE TITLE
check and correct table tags

### ISSUE THREAD

**Issue by mascazubi** [cunybpl/nycet_pnp#223]
[Accessibility test revealed](https://www.notion.so/nycsustainability/Design-Accessibility-Checklist-PnP-Website-107279213dc380138a9adff5f4bf39de) that tables might not have the proper tags for columns and rows and also may be missing closing tags for the various parts of the table element.

Need to go through all the tables and make corrections where needed.
